docs/running-on-yarn.md

```diff
@@ -604,3 +604,17 @@ spark.yarn.am.extraJavaOptions -Dsun.security.krb5.debug=true -Dsun.security.spn
 
 Finally, if the log level for `org.apache.spark.deploy.yarn.Client` is set to `DEBUG`, the log
 will include a list of all tokens obtained, and their expiry details
+
+## Using the Spark History Server to replace the Spark Web UI
+
+It is possible to use the Spark History Server application page as the tracking URL for running
+applications in scenarios where it may be desired to disable the built-in application UI. Two steps
```

Comment:
nit: first sentence reads a little funny.  maybe rephrase to:

It is possible to use the Spark History Server application page as the tracking URL for running
 applications where built-in application UI is disabled.  This may be desirable on secure clusters or to avoid the memory usage on the driver from the UI.

Up to you.  maybe doens't even need the second sentence.